README.markdown

```diff
@@ -163,6 +163,86 @@ can enforce the rendering of undefined members as `null`.
 (Note that this only affect JSON writing, spray-json will always read missing optional members as well as `null`
 optional members as `None`.)
 
+### Providing JsonFormats for enumeration
+
+There isn't an official support for enumeration types, but here a possible implementation example:
+
+```scala
+import spray.json._
+
+class EnumJsonFormat[T <: scala.Enumeration](enu: T) extends RootJsonFormat[T#Value] {
```

Comment:
Why the lib owners can't merge this `EnumJsonFormat` class to make it official? @davideicardi